syntax = "proto3";

package estimator;

message MachineInfo {
  int32 id = 1;
}

message RemainingLifetime {
  int32 id = 1;
  int32 lifetime = 2;
}

service LifetimeEstimator {
  rpc estimateRemainingLifetime(MachineInfo) returns (RemainingLifetime) {}
}